owncloudComLibrary/build.gradle

```diff
@@ -5,7 +5,6 @@ apply plugin: 'kotlin-kapt'
 apply plugin: 'kotlin-allopen'
 
 dependencies {
-    api 'com.squareup.okhttp3:okhttp:3.12.0'
```

Comment:
@davigonz But anyway, I would remove this misleading and pointless dependency and merge this PR